server/service/fleet_vars.go

```diff
@@ -0,0 +1,109 @@
+package service
```

Comment:
Can we decouple this from service package? Maybe `variables` package? Or maybe this is too small to bother?

Reply:
Would be happy to do that as well, since it doesn't actually interact with any I/O. I could also move it to `fleet`

Reply:
Ok, let's do `fleet`